src/gobprepare/utils/sql.py

```diff
@@ -0,0 +1,86 @@
+# SQL constants
+SQL_TYPE_CONVERSIONS = {
+    "String": "character varying",
+    "DateTime": "timestamp without time zone",
+    "Int": "integer",
+    "Boolean": "boolean",
+    "GeoJSON": "geometry",
+}
+SQL_QUOTATION_MARK = "'"
+
+
+def _quote(name):
+    """
+    Quote all SQL identifiers (schema, table, column names)
+    to prevent weird errors with SQL keywords accidentally being used in identifiers.
+
+    Note that quotation marks may differ per database type.
+    Current escape char works for PostgreSQL
+
+    :param name:
+    :return:
+    """
+    QUOTE_CHAR = '"'
+    return f"{QUOTE_CHAR}{name}{QUOTE_CHAR}"
+
+
+def _create_field(name, type, description):
+    """
+    Create a database field
+
+    :param name:
+    :param type:
+    :param description:
+    :return: dict containing database field properties
+    """
+    return {
+        'name': _quote(name),
+        'type': SQL_TYPE_CONVERSIONS[type],
+        'description': description
+    }
+
+
+def get_create_table_sql(schema, table_name, description, meta_fields, field_names):
+    """
+    Returns a SQL statement to create a table in a schema
+    The table fields are constructed from GraphQL meta fields
+
```

Comment:
The first field is the primary key. Is this somewhere related to the format or order of the GraphQL query? If so then this should be documented with the respective query.

Reply:
A primary key isn't even necessary. Logic to decide on primary key is based on implicit knowledge and assumptions. Why not just forget about the primary key? This way it can never go wrong, and keeps the code much simpler.

Reply:
fixed